auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -234,8 +233,6 @@ public FileSystemSecurityRealm(final Path root, final NameRewriter nameRewriter,
         this(root, nameRewriter, levels, encoded, Encoding.BASE64, StandardCharsets.UTF_8, INSTALLED_PROVIDERS, null);
     }
 
-
```

Comment:
It would be good to remove the white space changes.

Reply:
Removed